hoomd/hpmc/compute.py

```diff
@@ -311,16 +311,32 @@ def sdf(self):
         self._cpp_obj.compute(self._simulation.timestep)
```

Comment:
Rename `sdf` to `sdf_compression`. Create a deprecated `sdf` that aliases it. Document that deprecation in `deprecated.rst`.